src/main/java/com/ldtteam/blockout/controls/ToggleButton.java

```diff
@@ -44,8 +76,32 @@ public ToggleButton(final PaneParams params)
         {
             button.setText(states.get(active));
         }
+        else
+        {
+            button.clearText();
+        }
+    }
+
+    /**
+     * @param raw true if the result should not be localized
+     * @return the list of states as strings
+     */
+    public List<String> getStateStrings(boolean raw)
+    {
+        return raw ? rawStates : states.stream().map(IFormattableTextComponent::getString).collect(Collectors.toList());
     }
 
+    public List<IFormattableTextComponent> getStates()
```

Comment:
jdoc

Reply:
We haven't been adding jdoc for pure getters like this one. It wouldn't really clarify anything further. Between all the getters around that's hundreds of lines of code we don't really need. There was a brief discussion on this in the original PR. This is what I said then:
> I just imagine it as if it is in an auto-genned doc page. would someone understand that this sets the text alignment based off of the method title alone? If the answer is yes then it doesn't need a jdoc IMO. You just need to be careful that you aren't doing other things in the setter that won't be described by the title alone. 
But if you're just going to chuck ``/** Sets the text alignment */`` above it then that is not worth anyone's energy scrolling past it.